app-accessibility/flite/flite-2.1.recipe

```diff
@@ -0,0 +1,143 @@
+	cmd:flite_cmu_us_awb = $portVersion
+	cmd:flite_cmu_us_kal = $portVersion
+	cmd:flite_cmu_us_kal16 = $portVersion
+	cmd:flite_cmu_us_rms = $portVersion
+	cmd:flite_cmu_us_slt = $portVersion
+	cmd:flite_time$secondaryArchSuffix = $portVersion
+	lib:libflite$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_grapheme_lang$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_grapheme_lex$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_indic_lang$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_indic_lex$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_time_awb$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_us_awb$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_us_kal$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_us_kal16$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_us_rms$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmu_us_slt$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_cmulex$secondaryArchSuffix = $libVersionCompat
+	lib:libflite_usenglish$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	flite${secondaryArchSuffix}_devel = $portVersion
+	devel:libflite$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_grapheme_lang$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_grapheme_lex$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_indic_lang$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_indic_lex$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_time_awb$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_us_awb$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_us_kal$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_us_kal16$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_us_rms$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmu_us_slt$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_cmulex$secondaryArchSuffix = $libVersionCompat
+	devel:libflite_usenglish$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	flite$secondaryArchSuffix == $portVersion base
+	haiku${secondaryArchSuffix}_devel
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:autoconf
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+defineDebugInfoPackage flite$secondaryArchSuffix \
+	$binDir/flite_cmu_time_awb \
+	$binDir/flite_cmu_us_awb \
+	$binDir/flite_cmu_us_kal \
+	$binDir/flite_cmu_us_kal16 \
+	$binDir/flite_cmu_us_rms \
+	$binDir/flite_cmu_us_slt \
+	$binDir/flite_time \
+	$libDir/libflite.so.2 \
+	$libDir/libflite_cmu_indic_lang.so.2 \
+	$libDir/libflite_cmu_indic_lex.so.2 \
+	$libDir/libflite_cmu_time_awb.so.2 \
+	$libDir/libflite_cmu_us_awb.so.2 \
+	$libDir/libflite_cmu_us_kal16.so.2 \
+	$libDir/libflite_cmu_us_slt.so.2 \
+	$libDir/libflite_cmulex.so.2 \
+	$libDir/libflite_usenglish.so.2
```

Comment:
libVersion is not 2

Reply:
libflite_usenglish.so.2 and the others are no files ...

Reply:
They seem to appear in the "Contents" section of HaikuDepot, as well as the Makefile on the root directory of flite.